File: Anul 2/Semestrul 2/MPP/labs/Tema 1-proiect Java/proiect/src/main/java/com/example/proiect/model/Bilet.java
package com.example.proiect.model;

public class Bilet extends Entity<Long>{
    private String cumparatorName;
    private Long idSpectacol;
    private int nrSeats;

    public Bilet(String name, Long id_spectacol, int nrSeats)
    {
        this.cumparatorName=name;
        this.idSpectacol=id_spectacol;
        this.nrSeats=nrSeats;
    }

    public String getCumparatorName() {
        return cumparatorName;
    }

    public void setCumparatorName(String cumparatorName) {
        this.cumparatorName = cumparatorName;
    }

    public Long getIdSpectacol() {
        return idSpectacol;
    }

    public void setIdSpectacol(Long id_spectacol) {
        this.idSpectacol = id_spectacol;
    }

    public int getNrSeats() {
        return nrSeats;
    }

    public void setNrSeats(int nrSeats) {
        this.nrSeats = nrSeats;
    }
}

File: Anul 2/Semestrul 2/MPP/labs/Tema 2-proiect Csharp/proiect/proiect/model/Bilet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace proiect.model
{
    public class Bilet : Entity<long>
    {
        public string cumparatorName { get; set; } 
        public long id_spectacol { get; set; }  
        public int nrSeats { get; set; }

        public Bilet(string CumparatorName, long Id_spectacol, int NrSeats)
        {
            cumparatorName = CumparatorName;
            id_spectacol = Id_spectacol;
            nrSeats = NrSeats;
        }
        public override string ToString()
        {
            return cumparatorName + " | " + id_spectacol + " | " + nrSeats ;
        }
    }
}
